### ISSUE TITLE
Private exports shows as "public" when imported without an activation key

### ISSUE THREAD

**Issue by goku321** [nats-io/nsc#477]
Steps to reproduce:
1. Create an account A and a private export E
2. Create an account B and import E from A without an activation key (see attachments below)

Importing without an activation key shows no error but the import is listed as public when using `nsc describe account` command.

Private Export:

![Screenshot 2022-04-06 at 11 42 54 AM](https://user-images.githubusercontent.com/12848015/161907693-bfb97bc4-18bb-4888-b738-1669376d0edf.png)

After Importing:

![Screenshot 2022-04-06 at 11 42 38 AM](https://user-images.githubusercontent.com/12848015/161907782-486ce9ae-42d6-400c-95e9-e4600156a57c.png)

**Comment by aricart** [comment]
@goku321 how did you add the import? - did the tool walk you through it, or you entered manually?

**Comment by goku321** [comment]
@aricart I did the import manually

**Comment by aricart** [comment]
Did you generate and specify a token?

**Comment by matthiashanel** [comment]
What is being displayed is based on the jwt itself, no cross validation happens.
This is because you don't have to have the referenced account jwt in your store.
Since you did not set a token, it will read as public import.

**Comment by goku321** [comment]
> Did you generate and specify a token?

No. When I generate and specify the activation token, export is displayed as private which is expected

**Comment by goku321** [comment]
> What is being displayed is based on the jwt itself, no cross validation happens. This is because you don't have to have the referenced account jwt in your store. Since you did not set a token on import, it will read as public import.

If it will be read as a public import, Would a user be able to do a pub/sub without the activation token? (I didn't test it yet)

**Comment by matthiashanel** [comment]
> If it will be read as a public import, Would a user be able to do a pub/sub without the activation token? (I didn't test it yet)
No, if the exporting account requests a token, then the importing account has to provide it.
